09_regexp.md

```diff
@@ -25,137 +22,98 @@ if}}
 
 {{index evolution, adoption, integration}}
 
-Programming ((tool))s and techniques survive and spread in a chaotic,
-evolutionary way. It's not always the pretty or brilliant ones that
-win but rather the ones that function well enough within the right
-niche or that happen to be integrated with another successful piece of
-technology.
+经年累月，编程（工具）和技术在一种混乱的、优胜劣汰的方式中得以生存和普及。最终胜出的往往并非总是美观或出彩的技术，而是那些在小范围内非常实用、或是与其他良好的技术很契合的技术。
```

Comment:
can you keep the original `(( ))` bracket instead of using the chinese bracket please?